packages/api/src/resolvers/update/index.ts

```diff
@@ -39,6 +39,7 @@ export const updatePageResolver = authorized<
     id: input.pageId,
     title: input.title || undefined,
     description: input.description || '',
+    author: input.byline ?? undefined,
```

Comment:
I wonder, if old clients call this API, they will be sending undefined, so if the author is already set, would it then get set to undefined?

Reply:
no, it won't

Reply:
oh because it would be undefined and removed from what is being set. Thanks!

Reply:
yeah